src/Tests/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -121,6 +124,27 @@ public TestAsset UpdateProjProperty(string propertyName, string variableName, st
             }, targetValue);
         }
 
+        public TestAsset ToReplaceTheNewtonsoftJsonPackageVersionVariable()
```

Comment:
nit: I'd remove the To in front so the action is the first word
```suggestion
        public TestAsset ReplaceTheNewtonsoftJsonPackageVersionVariable()
```